identity-iota/src/chain/document_chain.rs

```diff
@@ -17,8 +18,8 @@ use crate::tangle::MessageId;
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
Add a short doc comment?

Reply:
```
/// Holds an [`IntegrationChain`] and its corresponding [`DiffChain`] that can be used to resolve the 
/// latest version of an [`IotaDocument`].
```

Updated; thanks for the suggested changes!